kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -478,6 +547,24 @@ function getMinReplicas {
   eval $__result=${minReplicas}
```

Comment:
Like the other commands, minReplicaCmd & minReplicas should be declared local. Furthermore, they should be unique enough so that there's potential collision with $__result...